src/app/services/sds.service.spec.ts

```diff
@@ -172,7 +172,7 @@ describe('SdsService', () => {
         'An error occurred:',
         'ErrorMessage'
       );
-      expect(result._subscribe).toBeTruthy();
+      expect(result.subscribe).toBeTruthy();
```

Comment:
I can't remember why I originally used the _subscribe property here, that's an internal property that shouldn't normally be used. I basically wanted to check that the return value is a subscribable object, so this update seems like an improvement. I may have been trying to check the underlying object rather than the subscribe helper method, but the old code seems odd to me now.